error TS2305: Module '"/app-project/node_modules/ng-zorro-antd/antd"' has no exported member 'NzTableComponent'.

## Version

0.7.1

## Environment

ng-zorro 0.7.1

## Reproduction link

[https://stackblitz.com/edit/angular-n9nkkn?file=src%2Fapp%2Ftable-fit.directive.ts](https://stackblitz.com/edit/angular-n9nkkn?file=src%2Fapp%2Ftable-fit.directive.ts)

## Steps to reproduce

1.ng build
2.报错如下
ERROR in src/app/core/directive/nz-table-fit.directive.ts(4,10): error TS2305: Module '"/Users/sunpeitao/mycodenew/zeus-app-project-static/node_modules/ng-zorro-antd/antd"' has no exported member 'NzTableComponent'.

## What is expected?

正常使用类 NzTableComponent

## What is actually happening?

ERROR in src/app/core/directive/nz-table-fit.directive.ts(4,10): error TS2305: Module '"/Users/sunpeitao/mycodenew/zeus-app-project-static/node_modules/ng-zorro-antd/antd"' has no exported member 'NzTableComponent'.

## Other?



<!-- generated by ng-zorro-issue-helper. DO NOT REMOVE -->

--- a/components/table/public-api.ts
+++ b/components/table/public-api.ts
@@ -1,4 +1,4 @@
-export * from './nz-thead.component';
+export * from './nz-table.component';
 export * from './nz-table.module';
 export * from './nz-tbody.directive';
 export * from './nz-td.component';

